unicore/ask/service/responses_api.py

```diff
@@ -0,0 +1,53 @@
+from cornice.resource import resource, view
+from pyramid.exceptions import NotFound
+
+from unicore.ask.service.models import QuestionResponse, Question
+
+
+def get_response_object(request):
+    uuid = request.matchdict['uuid']
+    response = request.db.query(QuestionResponse).get(uuid)
+
+    if response is None:
+        raise NotFound
+
+    return response
+
+
+def get_responses(request):
+    question_uuid = request.matchdict['question_uuid']
+
+    question = request.db.query(Question).get(question_uuid)
+
+    if question is None:
+        raise NotFound
+
+    return request.db.query(QuestionResponse).filter_by(
+        question_id=question_uuid).order_by('question_id')
```

Comment:
It would be better to add a `responses` relationship to the `Question` model, like you did for `QuestionOption` [here](https://github.com/universalcore/unicore.ask/blob/feature/issue-3-add-get-api-for-responses/unicore/ask/service/models.py#L55). Then use `question.responses.order_by('question_id')`.